Makefile

```diff
@@ -242,7 +242,7 @@ $(ENVTEST): $(LOCALBIN)
 .PHONY: golangci-lint
 golangci-lint: $(GOLANGCI_LINT) ## Download golangci-lint locally if necessary.
 $(GOLANGCI_LINT): $(LOCALBIN)
-	@GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on $(GO_CMD) install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION)
+	@GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on $(GO_CMD) install github.com/golangci/golangci-lint/v2/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION)
```

Comment:
can we also bump GH action to `golangci/golangci-lint-action@v7`

Reply:
@shivamerla Its already at `golangci/golangci-lint-action@v8`

Reply:
Also, add a .golangci.yaml similar to [here](https://github.com/NVIDIA/k8s-dra-driver-gpu/blob/main/.golangci.yaml) for all linter checks.